src/main/java/com/cloudbees/jenkins/support/actions/SupportObjectAction.java

```diff
@@ -141,12 +141,16 @@ public final void doGenerateAndDownload(StaplerRequest2 req, StaplerResponse2 rs
         try {
             SupportPlugin.setRequesterAuthentication(Jenkins.getAuthentication2());
             try (ACLContext old = ACL.as2(ACL.SYSTEM2)) {
-                SupportPlugin.writeBundle(rsp.getOutputStream(), components, new ComponentVisitor() {
-                    @Override
-                    public <C extends Component> void visit(Container container, C component) {
-                        ((ObjectComponent<T>) component).addContents(container, object);
-                    }
-                });
+                SupportPlugin.writeBundle(
+                        rsp.getOutputStream(),
+                        components,
+                        new ComponentVisitor() {
+                            @Override
+                            public <C extends Component> void visit(Container container, C component, double progress) {
+                                ((ObjectComponent<T>) component).addContents(container, object);
+                            }
+                        },
+                        null);
```

Comment:
Note that these per-component **Support** links do _not_ use background generation. Probably fine; these are typically very quick to run.